plugins/acpi-wpbt/README.md

```diff
@@ -0,0 +1,15 @@
+ACPI WPBT
+=========
+
+Introduction
+------------
+
+This plugin checks if the WPBT table has been exported. Although unused by
+Linux, Microsoft Windows will unconditionally execute the EXE file specified by
+the WPBT table. WPBT should be disabled in the firmware setup menu, and it
+certainly should not be enabled by default at the factory.
+
+This is used by CompuTrace for legitimate hardware recovery and less commonly
+to install OEM-specific drivers and utilities on a "clean" windows tool.
```

Comment:
Since this has a legitimate use, I feel that we need a user who has chosen to enable something like computrace for their threat model to be able opt out of this being a runtime error.

Reply:
`BlacklistPlugins=`, no?

Reply:
Yes, I think that is a sufficient lever. We should probably have it mentioned in the readme though to discuss your threat model how to actually do it.

Reply:
I've added a chunk to the README.